src/Makefile.am

```diff
@@ -161,6 +161,7 @@ source_h = \
         i965_encoder_common.h        \
         i965_encoder_api.h        \
         gen9_avc_const_def.h      \
+        gen9_avc_encoder_kernels.h      \
```

Comment:
The kernels should be using the APACHE 2 license and NOT the MIT license.  Please see VP8 patch series comments and subsequent changes by @xhaihao .  Same rule applies for AVC kernels.